docs/reference/commandline/secret_ls.md

```diff
@@ -36,8 +38,78 @@ Run this command on a manager node to list the secrets in the swarm.
 ```bash
 $ docker secret ls
 
-ID                          NAME                    CREATED                                   UPDATED
-mhv17xfe3gh6xc4rij5orpfds   secret.json             2016-10-27 23:25:43.909181089 +0000 UTC   2016-10-27 23:25:43.909181089 +0000 UTC
+ID                          NAME                        CREATED             UPDATED
+6697bflskwj1998km1gnnjr38   q5s5570vtvnimefos1fyeo2u2   6 weeks ago         6 weeks ago
+9u9hk4br2ej0wgngkga6rp4hq   my_secret                   5 weeks ago         5 weeks ago
+mem02h8n73mybpgqjf0kfi1n0   test_secret                 3 seconds ago       3 seconds ago
+```
+### Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* [id](secret_ls.md#id)
+* [label](secret_ls.md#label)
+* [name](secret_ls.md#name)
+* [names](secret_ls.md#names)
+
+#### id
+
+The `id` filter matches all or part of a secret's id.
```

Comment:
Hm, I think we do this in other places as well, but wondering; do we actually match a _part_ or only a _prefix_ (which is, erm, a part, but only if the ID _starts with_ the given value)